RiotSharp/MatchEndpoint/TeamStats.cs

```diff
@@ -83,15 +94,15 @@ public class Team
         public int TowerKills { get; set; }
 
         /// <summary>
-        /// Number of times the team killed vilemaw.
+        /// Number of times the team killed vilemaw (Twisted Treeline epic monster).
         /// </summary>
         [JsonProperty("vilemawKills")]
         public int VilemawKills { get; set; }
 
         /// <summary>
         /// Flag indicating whether or not the team won.
```

Comment:
I don't think the doc matches anymore

Reply:
Why are you saying that ?

Reply:
because it's not a flag anymore since it's a string

Reply:
Oh the review was for the flag. It was not clear because the "centered" lines were the ones about Vilemaw.

Reply:
yes I try to comment about the line just above :+1: